endpoints/openrtb2/auction.go

```diff
@@ -52,7 +52,7 @@ import (
 	"github.com/prebid/prebid-server/version"
 )
 
-const storedRequestTimeoutMillis = 50
+const storedRequestTimeoutMillis = 500
```

Comment:
Changing this time might have negative consequences for some hosts. I recommend either making this change in a fork or altering this PR to make it a configuration. I vaguely remember a PR a while back to make it configurable where I think the author lost interest. Happy if you'd like to pick that back up.

Reply:
Hi @GLStephen, just curious what your thoughts are here. Thanks!